apps/frontend/src/libs/components/app/app.tsx

```diff
@@ -1,3 +1,5 @@
+import { useCallback, useState } from "react";
```

Comment:
see how we improv hooks in the rest of the application. always follow the style of the app